packages/ethereum/cardstack/event-indexer.js

```diff
@@ -64,13 +62,12 @@ class EthereumEventIndexer {
   async _indexRecord(batch, record) {
     log.debug('indexing model in pgsearch %j', record);
     let { id, type } = record;
-    let schema = await this.schema.forControllingBranch();
+    let schema = await this.currentSchema.getSchema();
```

Comment:
@robustdj I'm unsure if `current-schema` is injected into any of the modules we use in portfolio, but it it is, this has been updated to return a schema from the `getSchema()` function. (this used to only get a schema for a specific branch)